app/src/main/res/values/styles_v2.xml

```diff
@@ -167,4 +175,14 @@
     <item name="android:src">@drawable/ic_clear_circular_24dp</item>
     <item name="android:layout_gravity">end</item>
   </style>
+
+  <style name="Clinic.V2.OverdueListActionItems">
```

Comment:
This style is for an individual action item so you can just name it as `OverdueListActionItem`.

Reply:
Actually, there's no need to suffix it with `Item` either. Just `OverdueListAction` is sufficient.